### PR TITLE
Update overview.md

### PR DESCRIPTION
**Pull request by billgib**

### CODE REVIEW

File: documentation/overview.md [hunk 1]
```diff
@@ -314,77 +320,93 @@ offset must be applied when needed to compute local time.
...
 
 The following features are not yet supported:
-- Use of headers when reading or writing CSV files and using a separator character other than a comma.
-- Use and configuration of partition patterns for organizing data files on write.
-- Nested parquet support.
-- In explicit write, use of a distinct ADLS storage account for the entity definition. Initially, the
-entity definition must be in the same storage account as the target CDM folder.
+- Overriding a timestamp column to be interpreted as a CDM Time rather than a DateTime is initially supported for CSV files only.  Support for writing Time data to Parquet will be added in a later release.
+- In explicit write, the container for the entity definition must be in the same ADLS storage account as the target CDM folder.  Support for the entity definition being in a different storage account will be added in a later release.
+- Parquet Maptype and arrays of primitive types are not currently supported by CDM so are not supported by the Spark CDM Connector.
```
Comment (sricheta92) [inline documentation/overview.md:408]: Do you want to specify that ArrayType of ArrayTypeis not supported too..

File: documentation/overview.md [hunk 2]
```diff
@@ -260,41 +270,38 @@ df.write.format("com.microsoft.cdm")
...
  .option("storage", "mystorage.dfs.core.windows.net")
  .option("container", "cdmdata")
- .option("manifest", "/contacts/root.manifest.cdm.json")
- .option("entity", "Person")
- .option("useCDMGithub", true)
- .option("appId", appid)
- .option("appKey", appkey)
- .option("tenantId", tenantid)
- .load()
+ .option("manifest", "Teams/root.manifest.cdm.json")
```
Comment (sricheta92) [inline documentation/overview.md:285]: Remove container and manifest option and replace with manifestPath - cdmdata/Teams/root.manifest.cdm.json

File: documentation/overview.md [hunk 3]
```diff
@@ -260,41 +270,38 @@ df.write.format("com.microsoft.cdm")
...
+ .option("entity", "TeamMembership")
+ .option("entityDefinition", "core/applicationCommon/TeamMembership.cdm.json/Tea
```
Comment (sricheta92) [inline documentation/overview.md:287]: entityDefinition -> entityDefinitionPath

File: documentation/overview.md [hunk 4]
```diff
@@ -260,41 +270,38 @@ df.write.format("com.microsoft.cdm")
...
+mMembership")
+ .option("useCdmGithubModelRoot", true)
```
Comment (sricheta92) [inline documentation/overview.md:289]: useCdmGithubModelRoot -> useCdmStandardModelRoot

File: documentation/overview.md [hunk 5]
```diff
@@ -260,41 +270,38 @@ df.write.format("com.microsoft.cdm")
...
- .option("tenantId", tenantid)
  .mode(SaveMode.Overwrite)
  .save()
 ```
+#### Explicit Write - using an entity defined in the CDM GitHub
 
-#### Read
-
-This code reads the Person entity from the CDM folder with manifest at [https://mystorage.dfs.core.windows.net/cdmdata/contacts/root.manifest.cdm.json](https://mystorage.dfs.core.windows.net/cdmdata/contacts/root.manifest.cdm.json)
+This code writes the dataframe _df_ to a CDM folder with the manifest at [https://mystorage.dfs,core.windows.net/cdmdata/Teams/root.manifest.cdm.json](https://mystorage.dfs,core.windows.net/cdmdata/Teams/root.manifest.cdm.json) and a sub-manifest containing the TeamMembership entity, created in a TeamMembership subdirectory. TeamMembership data is written as CSV files (by default) that overwrite any existing data files. The entity definition is retrieved from the CDM GitHub repo, at:
```
Comment (sricheta92) [inline documentation/overview.md:278]: The entity definition is retrieved from the CDM _CDN location_?

File: documentation/overview.md [hunk 6]
```diff
@@ -260,41 +270,38 @@ df.write.format("com.microsoft.cdm")
  .option("entityDefinitionContainer", "cdmmodels")
```
Comment (sricheta92) [inline documentation/overview.md:270]: You might want to remove entityDefinitionContainer and update entityDefinitionModelRoot -> cdmmodels/core

File: documentation/overview.md [hunk 7]
```diff
@@ -55,153 +56,188 @@ The following scenarios are not supported:
...
 
 The following examples all use appId, appKey and tenantId variables initialized earlier in the code based on an Azure app registration that has been given Storage Blob Data Contributor permissions on the storage for write and Storage Blob Data Reader permissions for read.
 
+#### Read
+This code reads the Person entity from the CDM folder with manifest in mystorage.dfs.core.windows.net/cdmdata/contacts/root.manifest.cdm.json.
+
+```scala
+val df = spark.read.format("com.microsoft.cdm")
+ .option("storage", "mystorage.dfs.core.windows.net")
+ .option("container", "cdmdata")
```
Comment (sricheta92) [inline documentation/overview.md:233]: Same as below comments

Remove container and manifest option and replace with manifestPath - cdmdata/contacts/root.manifest.cdm.json

File: documentation/overview.md (hunk 1 above)
Comment (sricheta92) [inline documentation/overview.md:408]: Do you want to add Databricks not able to use the CDN location issue here?

File: documentation/overview.md [hunk 8]
```diff
@@ -55,153 +56,188 @@ The following scenarios are not supported:
...
-|useCdmGithub|Sets the modelroot alias to [https://github.com/microsoft/CDM/tree/master/schemaDocuments](https://github.com/microsoft/CDM/tree/master/schemaDocuments) to resolve the reference to foundations.cdm.json. <br/>**Required for Implicit Write and Read  only.** |"useCdmGithub" |
-|*entityDefinitionStorage [NOT YET SUPPORTED]*|*The ADLS gen2 storage account containing the entity definition. Required if different to the storage account hosting the CDM folder.*|*\<accountName\>.dfs.core.windows.net*|*"[myAccount.dfs.core.windows.net]"*|
-|entityDefinitionContainer|The storage container containing the entity definition. Required if different to the CDM folder container.| \<containerName><br/> "models"|
+|*entityDefinitionStorage [NOT YET SUPPORTED]*|*The ADLS gen2 storage account containing the entity definition. Required if different to the storage account hosting the CDM folder.*|*\<accountName\>.dfs.core.windows.net*<br/>_"myAccount.dfs.core.windows.net"_|
+|entityDefinitionContainer|The storage container/file system containing the entity definition. Required if different to the CDM folder container.| \<containerName><br/> "models"|
```
Comment (sricheta92) [inline documentation/overview.md:177]: I think we want to remove this option now - entityDefinitionContainer ?

File: documentation/overview.md [hunk 9]
```diff
@@ -55,153 +56,188 @@ The following scenarios are not supported:
...
 |entityDefinitionModelRoot|The location of the model root or corpus within the container. If not specified, defaults to the root folder.|\<folderPath\> <br/> "crm/core"<br/> (use "/" for the root folder of the container)|
```
Comment (sricheta92) [inline documentation/overview.md:178]: entityDefinitionModelRoot -> container/<folder_path>

File: documentation/overview.md [hunk 10]
```diff
@@ -55,153 +56,188 @@ The following scenarios are not supported:
...
 |entityDefinition|Location of the entity. File path to the CDM definition file relative to the model root, including the name of the entity in that file.|\<folderPath\>/\<entityName\>.cdm.json/\<entityName\><br/>"sales/customer.cdm.json/customer"|
```
Comment (sricheta92) [inline documentation/overview.md:179]: entityDefinition -> entityDefinitionPath

File: documentation/overview.md [hunk 11]
```diff
@@ -55,153 +56,188 @@ The following scenarios are not supported:
...
 |---------|---------|:---------:|
-|storage|An ADLS gen2 storage account *with HNS enabled* in which the CDM folder is located.  Be sure to use the dfs.core.windows.net URL | \<accountName\>.dfs.core.windows.net "myAccount.dfs.core.windows.net"        |
-|container|An ADLS gen2 file system / container in which the source or target CDM folder is located | \<containerName\> <br/>"myContainer"        |
-|manifest|The relative path to the manifest or model.json file from the container root. For read, can be a root manifest or a sub-manifest. For write, must be the root manifest.|[\<folderPath\>]/fullManifestName, "default.manifest.cdm.json" "employees.manifest.cdm.json" <br/> "employees/person.manifest.cdm.json" "employees/model.json" (read only)         |
+|storage|The endpoint URL for then ADLS gen2 storage account *with HNS enabled* in which the CDM folder is located.  <br/>Use the **dfs**.core.windows.net URL | \<accountName\>.dfs.core.windows.net "myAccount.dfs.core.windows.net"        |
+|container|An storage container/file system in which the source or target CDM folder is located | \<containerName\> <br/>"myContainer"        |
```
Comment (sricheta92) [inline documentation/overview.md:163]: remove

File: documentation/overview.md [hunk 12]
```diff
@@ -55,153 +56,188 @@ The following scenarios are not supported:
...
+|manifest|The relative path to the manifest or model.json file from the container root. For read, can be a root manifest or a sub-manifest. For write, must be the root manifest.|[\<folderPath\>]/fullManifestName, <br/>"default.manifest.cdm.json" "employees.manifest.cdm.json" <br/> "employees/person.manifest.cdm.json" <br/>"employees/model.json" (read only)         |
```
Comment (sricheta92) [inline documentation/overview.md:164]: manifest -> manifestPath..

File: documentation/overview.md [hunk 13]
```diff
@@ -40,13 +38,16 @@ The following scenarios are supported:
...
 - Supports reading from CDM folders described by either manifest or model.json files.
-- Supports writing to CDM folders described by a manifest file only. Write support for model.json file is not planned.
-- Supports data in CSV and Apache Parquet format. Support for nested parquet files is planned.
-- Supports partition patterns on read and sub-manifests on read and write.
+- Supports writing to CDM folders described by a manifest file. Write support for model.json file is not planned.
+- Supports data in CSV format with/without column headers and with user selectable delimiter character   
+- Supports data in Apache Parquet format, including nested parquet.
+- Supports sub-manifests on read, optional use of entity-scoped submanifests on write.
+- Supports writing data using user modifiable partition patterns.
+- Supports use of managed identity (Synapse), user identity (Azure Databricks) and credentials
```
Comment (sricheta92) [inline documentation/overview.md:48]: Do we want to add supports reading config.json?

File: documentation/overview.md [hunk 14]
```diff
@@ -264,29 +262,26 @@ The entity definition is retrieved from
...
-[https://github.com/microsoft/CDM/tree/master/schemaDocuments/core/applicationCommon/TeamMembership.cdm.json/TeamMembership](https://github.com/microsoft/CDM/tree/master/schemaDocuments/core/applicationCommon/TeamMembership.cdm.json/TeamMembership)
+This code writes the dataframe _df_ to a CDM folder with the manifest at [https://mystorage.dfs,core.windows.net/cdmdata/Teams/root.manifest.cdm.json](https://mystorage.dfs,core.windows.net/cdmdata/Teams/root.manifest.cdm.json) and a sub-manifest containing the TeamMembership entity, created in a TeamMembership subdirectory. TeamMembership data is written as CSV files (by default) that overwrite any existing data files. The entity definition is retrieved from the CDM CDN, at:
+[https://cdm-schema.microsoft.com/logical//schemaDocuments/core/applicationCommon/TeamMembership.cdm.json/TeamMembership](https://cdm-schema.microsoft.com/logical//core/applicationCommon/TeamMembership.cdm.json/TeamMembership)
```
Comment (sricheta92) [inline documentation/overview.md:275]: is this URL correct?

Comment (billgib) [reply documentation/overview.md:275]: Nope - fixed both parts, incl. removing dangling reference to schema documents

**Pull request merged**